pyproject.toml

```
@@ -51,6 +51,7 @@ moderngl-window = "^2.3.0"
 mapbox-earcut = "^0.12.10"
 cloup = "^0.7.0"
 requests = "*"
+dearpygui = "^0.6.415"
```

I would like to have this optional. I could see there are checks for this already in py files. Maybe add this as an extra which can be installed using `pip install manim[gui]`?

---

I added a `--enable_gui` flag.

---

What I meant was making it optional while installing like how webgl_renderer is ie.
```suggestion
dearpygui = { version = "^0.6.415", optional = true }
```